README.md

```diff
@@ -1,3 +1,5 @@
+This is a fork of https://github.com/not-jan/pass-ssh
```

Comment:
Can you drop this from the PR? Doesn't really make sense when I merge it back in :)

Reply:
My bad, first time doing a PR.